pkg/deploy/generator/templates.go

```diff
@@ -0,0 +1,221 @@
+package generator
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	mgmtkeyvault "github.com/Azure/azure-sdk-for-go/services/keyvault/mgmt/2016-10-01/keyvault"
+
+	"github.com/Azure/ARO-RP/pkg/util/arm"
+)
+
+func (g *generator) _managedIdentityTemplate() *arm.Template {
```

Comment:
please remove the leading `_` from all of these functions.

Reply:
If I do this I have to rename each function in `generators.go` with different name:
```
sharedDevelopmentEnvTemplate(file){
   _sharedDevelopmentEnvTemplate()
   write file
```
If I rename those with `generateSharedDevelopmentEnvTemplate(file)` I get `generate.generateSharedDevelopmentEnvTemplate(file)` which is repetetive. I can't come up with new name for these which would not be repetetetive and nice, simple.